src/layout/shadowdom.ts

```diff
@@ -159,21 +168,24 @@ export class ShadowDom {
   }
 
   public mirrorsRealDom(): boolean {
```

Comment:
There isn't one, because i'm not aware of any exact scenarios right now that would lead to inconsistencies. However, looking at our data might help us figure out the root causes of the errors and whether they result from shadowDom arriving to the inconsistent state or something else.